### PR TITLE
Refactor website codebase

### PR DESCRIPTION
**Pull request by aayush-05**
Covered in this PR:
--Cleaned up code to use React-bootstrap via npm package (was using CDN previously)
--Moved styles to Sass
--Refactored current codebase
--Revamped few portions and made everything responsive

Currently left:
--Teams Page (to be done in a separate PR after finalising design)

PS: Not adding new sections (like Values, Timeline) right now >> Will be done once the codebase gets cleaned up

### CODE REVIEW

**Comment by jaskiratsingh2000** [comment]
Checks are failing

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@ node_modules/
 .DS_Store
 ngrok
 build/
-lambda/
\ No newline at end of file
+lambda/
```
Comment (vaibhavdaren) [inline .gitignore:5]: :)

File: package.json [hunk 2]
```diff
@@ -55,7 +56,7 @@
     "heroku-postbuild": "npm run build",
     "start": "serve -s build -l 3000",
     "prebuild": "netlify-lambda build netlify-functions",
-    "build": "CI=false && set \"CI=false\" && react-scripts build",
+    "build": "CI=false || set \"CI=false\" && react-scripts build",
```
Comment (vaibhavdaren) [inline package.json:59]: Does this treat errors as warnings or the opposite?

Comment (aayush-05) [reply package.json:59]: It does. On linux, the command is "CI=false" and on windows it is "set CI=false". We want one of them to run, not both.

File: .gitignore (hunk 1 above)
Comment (aayush-05) [reply .gitignore:5]: My prettier doing random things. Do you mean to remove this? Also should we add "package-lock.json" to .gitignore as well?

**Comment by aayush-05** [comment]
Ready to review :D

File: src/pages/Home/Components/Partners.js [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
+            alt="Women Who Code Logo"
           />
-          <img
-            src={gitbook}
-            alt="gitbook"
-            style={{ width: "15%", marginRight: 0 }}
+          <Image
+            className="backer-logo2 backer-gitbook-logo"
+            src={gitbookImage}
+            alt="Women Who Code Logo"
```
Comment (vaibhavdaren) [inline src/pages/Home/Components/Partners.js:46]:
```suggestion
            alt="Gitbook Image"
```

File: src/pages/Home/Components/Partners.js [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
-          <img
-            src={linuxFoundation}
-            alt="the linux foundation"
-            style={{ width: "15%", marginRight: 70 }}
+        </Col>
+        <Col lg={5} className="backer-logo-container">
+          <Image
+            className="backer-logo2"
+            src={linuxFoundationImage}
+            alt="Women Who Code Logo"
```
Comment (vaibhavdaren) [inline src/pages/Home/Components/Partners.js:41]:
```suggestion
            alt="Linux Foundation Image"
```

File: src/pages/Home/Components/Partners.js [hunk 5]
```diff
@@ -1,54 +1,54 @@
...
           />
-          <img
-            src={dgocean}
-            alt="digital ocean"
-            style={{ width: "15%", marginRight: 70 }}
+          <Image
+            className="backer-logo"
+            src={dgoceanImage}
+            alt="Women Who Code Logo"
```
Comment (vaibhavdaren) [inline src/pages/Home/Components/Partners.js:29]:
```suggestion
            alt="Digital Ocean image"
```

**Review by vaibhavdaren — APPROVED** [review]
Looks good.

**Pull request merged**